frontend/package.json

```diff
@@ -2,10 +2,20 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
+  "config": {
+    "input": "src/",
+    "intermediate": "tmp/",
+    "output": "dist/"
+  },
   "scripts": {
-    "start": "parcel serve src/!(layout).html --port 3000 --dist-dir .serve",
-    "build": "parcel build \"src/!(layout).html\"",
-    "lint": "eslint ."
+    "11ty:build": "eleventy --input=src/ --output=tmp/",
```

Comment:
Add this temp directory to the `gitignore` file. And remove it from Git before that. Also, I propose that we name it something like `.tmp`, as initial dot indicates a hidden folder which is good for a temporary container.

Reply:
Also, `npx @11ty/eleventy` is the recommended way of calling 11ty CLI.

Reply:
updated script:
 _"scripts": {
    "start": "run-p -l 11ty:watch parcel:serve",
    "11ty:build": "npx @11ty/eleventy",
    "11ty:watch": "npx @11ty/eleventy --watch",
    "build": "run-s -l clean 11ty:build parcel:build",
    "clean": "rm -rf dist/ _site/",
    "parcel:build": "parcel build \"./_site/**/*.html\" --dist-dir dist/",
    "parcel:serve": "parcel serve \"./_site/**/*.html\" --dist-dir dist/",
    "prestart": "run-s clean"
  },_

Reply:
Updated script:
 "scripts": {
    "start": "run-p -l 11ty:watch parcel:serve",
    "11ty:build": "npx @11ty/eleventy",
    "11ty:watch": "npx @11ty/eleventy --watch",
    "build": "run-s -l clean 11ty:build parcel:build",
    "clean": "rm -rf dist/ _site/",
    "parcel:build": "parcel build \"./_site/**/*.html\" --dist-dir dist/",
    "parcel:serve": "parcel serve \"./_site/**/*.html\" --dist-dir dist/",
    "prestart": "run-s clean"
  },